polaris.shopify.com/src/components/Markdown/Markdown.module.scss

```diff
@@ -0,0 +1,3 @@
+.MarkdownImage {
+  width: 100%;
```

Comment:
Because we're using the unwrap-images plugin, we need to set it to 100% width to match previous behaviour.